pkgs/top-level/aliases.nix

```diff
@@ -1197,6 +1197,22 @@ mapAliases {
   '';
   linux_latest_hardened = linuxPackages_latest_hardened;
 
+  # Added 2025-08-10
+  linuxPackages_hardened = linuxKernel.packages.linux_hardened;
```

Comment:
I think this will break `nixos/modules/profiles/hardened.nix` when `!enableAliases`; that should probably be fixed.

Another alternative would be to just [remove the profile](https://discourse.nixos.org/t/proposal-to-deprecate-the-hardened-profile/63081/5), which is probably overdue at this point…